### DISCUSSION TITLE
Github copilot

### DISCUSSION THREAD

**Discussion by Vigarista21** [community/community#106374] [GitHub Education]
### Hi everyone,

Então eu fui aprovado no github studant mas não recebi meus beneficios não sei oque fazer

**Comment by Fyphen1223** [comment]
In my experience, I had to wait 5 days to get my benefits since I was verified.
I recommend you to wait a while.

**Reply by Vigarista21** [reply]
então, eu ja tinha sido aprovado ano passado e não foram ativados meus benefícios
